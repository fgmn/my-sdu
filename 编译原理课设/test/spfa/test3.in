50
80
0
18
15
0
46
2
0
21
26
1
29
15
1
3
4
1
23
20
1
24
22
1
1
7
1
17
8
2
12
3
2
6
8
2
48
21
2
23
24
2
41
29
3
13
7
3
42
12
3
12
4
3
11
13
4
12
7
4
15
4
4
45
19
4
8
26
5
24
29
5
25
19
6
43
11
7
40
28
7
35
20
8
44
3
9
36
12
9
19
29
10
47
30
10
47
17
11
45
30
11
14
26
11
18
12
11
34
10
12
34
16
12
16
12
13
46
29
14
27
19
15
45
19
15
27
28
16
22
17
16
46
25
16
32
15
17
21
21
18
22
1
18
24
29
18
22
27
18
32
7
19
20
23
19
25
25
19
48
16
19
26
30
20
27
10
20
30
28
22
47
8
22
39
11
22
45
22
22
33
23
22
45
10
23
35
27
23
33
7
24
42
28
24
26
1
25
45
18
26
45
30
26
35
3
27
40
19
30
38
21
30
44
25
31
47
21
33
47
28
34
48
28
34
42
13
35
41
9
36
44
27
36
48
18
43
45
29
44
46
30
