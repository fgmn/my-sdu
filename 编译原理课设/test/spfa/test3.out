0
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
15
590558003
590558003
26
16
590558003
44
590558003
45
590558003
590558003
590558003
590558003
590558003
22
39
590558003
48
590558003
590558003
590558003
27
590558003
57
72
590558003
590558003
26
2
24
590558003
590558003
